server/chat.ts

```diff
@@ -2022,6 +2035,16 @@ export const Chat = new class {
 		return ' ' + message.toLowerCase() + ' ';
 	}
 
+	/**
+	 * JUST FOR HOTPATCHING. To be removed later.
```

Comment:
I thought about this - I don't think this actually can be removed later? `blockPMs: true` is stored in client's localStorage now, and old clients are going to still be sending this? I don't think its reasonable to remove this without a sufficiently long sunset period?